bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx

```diff
@@ -78,11 +78,7 @@ class ActionsBar extends PureComponent {
             : null
           }
         </div>
-        <div
-          className={
-            amIPresenter ? cx(styles.centerWithActions, actionBarClasses) : styles.center
-          }
-        >
+        <div className={cx(actionBarClasses)}>
```

Comment:
can remove the `cx` and also remove the import of cx

Reply:
Removing the `cx` breaks the styling 
![image](https://user-images.githubusercontent.com/22058534/69467453-09bdbe80-0d56-11ea-903f-d51adf5eebff.png)